### PR TITLE
refine doc and drg

### PR DESCRIPTION
**Pull request by bjoydeep**
@berenss a few more refinements for your review.

### CODE REVIEW

**Comment by berenss** [comment]
in the diagram, should we separate the apps and the policies as independent circles? or, safe to assume that they count linearly together. just wondering if we need to account for scenarios where
- customer uses tons of policies and almost 0 apps
- customer uses tons of apps and almost 0 policies
- customer uses tons of apps and tons of policies

File: README.md [hunk 1]
```diff
@@ -30,17 +30,17 @@ Well then you’ve come to the right place! We here aim to reveal, in a not so m
...
 
 So, to trace one line of the flow end to end:
-`API Server Object target count and size` drives load on the `ACM App & Policy Operators`. The `ACM App & Policy Operators` are also influenced by the `Cluster Count` - ie number of clusters - into which the applications and policies these have to be replicated to. These in turn creates resources on the `Kube API Server`. These resources are created in `etcd`. Therefore etcd health is one of the key drivers of `ACM Health`. And `etcd` health is also dependent on `Network health` and right `Disk health`.
+`Num of apps & policies` drives the `API Server Object target count and size` which in turn drives load on the `ACM App & Policy Operators`. The `ACM App & Policy Operators` are also influenced by the `Cluster Count` - ie number of clusters - into which the applications and policies these have to be replicated to. These in turn creates resources on the `Kube API Server`. These resources are created in `etcd`. Therefore etcd health is one of the key drivers of `ACM Health`. And `etcd` health is also dependent on `Network health` and right `Disk health`.
```
Comment (berenss) [inline README.md:44]: app & policy controllers?

Comment (berenss) [reply README.md:44]: And `etcd` health is also dependent on `Network health` and right `Disk health`.
And `etcd` health is also dependent on `Network health` and ~right~ `Disk health`.

File: README.md [hunk 2]
```diff
@@ -98,7 +98,32 @@ This has been tested using python 3.9.12. But if you do not want the hassle of s
...
+      |Postgres|Yes|
+      |Search Health|Can be inferred|
+      |Search API|Need to be added|
+      |ACM Health|Can be inferred|
+      |Cluster Count|Yes|
+      |Num of apps & policies|Yes|
+      |Time Series Count|Yes|
+      |Kubernetes Resource Count|Yes|
+
+3. `Current` data is printed out in the screen as below:
```
Comment (berenss) [inline README.md:127]: this is great!

File: README.md (hunk 1 above)
Comment (bjoydeep) [reply README.md:44]: changed both

**Comment by bjoydeep** [comment]
@berenss I did not add more to the drawing as it is already crowded - and probably we will need more drill downs per section  (app/policy/obs/search). But the over all representation - or most important variants are expressed here. That is the intent. I added this:
```
 For the sake of brevity this node represents both applications and policies. So this works if there is only applications, or only policies or both.
``` 
When this node is being explained. Hope this works.

**Pull request merged**